gradlew.bat

```diff
@@ -80,8 +80,8 @@ if "%ERRORLEVEL%"=="0" goto mainEnd
 :fail
 rem Set variable GRADLE_EXIT_CONSOLE if you need the _script_ return code instead of
 rem the _cmd.exe /c_ return code!
-if  not "" == "%GRADLE_EXIT_CONSOLE%" exit 1
-exit /b 1
+if  not "" == "%GRADLE_EXIT_CONSOLE%" exit %ERRORLEVEL%
```

Comment:
Nit: Mind getting rid of the double-space after `if` while at it? Similar below.

Reply:
Sorry, I just realized you're already doing this in the third commit 👍🏻